content/en/guide/v10/getting-started.md

```diff
@@ -225,27 +223,12 @@ These rewrites are configured using regular expressions in your Jest configurati
 }
 ```
 
-#### Aliasing in Snowpack
-
-To alias in [Snowpack](https://www.snowpack.dev/), you'll need to add a package import alias to the `snowpack.config.mjs` file.
```

Comment:
Snowpack has the following message on their site:

> Update (April 20, 2022): Snowpack is no longer actively maintained and is not recommended for new projects.

I think it's safe to remove now.